ports/dustaet/port.json

```
@@ -22,7 +22,7 @@
     "image": null,
     "rtr": false,
     "exp": false,
-    "runtime": [],
+    "runtime": "mono-6.12.0.122-aarch64.squashfs",
```

```suggestion
    "runtime": ["mono-6.12.0.122-aarch64.squashfs"],
```
Version 4 port files require "runtime" to be a list of strings instead of a single string.